app/services/uuid_reporter.rb

```diff
@@ -0,0 +1,123 @@
+class UuidReporter
+  def self.run(**opts)
+    new(**opts).run
+  end
+
+  def initialize(**options)
+    @emails = parse_file(options[:email_file]).map(&:downcase).uniq
+    validate_emails
+    @issuers = parse_file(options[:sp_file]).uniq
+    validate_issuers
+    @output = options[:output]
```

Comment:
let's make these keyword params:

```suggestion
  def initialize(email_file:, sp_file:, output:)
    @emails = parse_file(email_file).map(&:downcase).uniq
    validate_emails
    @issuers = parse_file(sp_file).uniq
    validate_issuers
    @output = output
```

Reply:
good catch, thanks!